### PR TITLE
fix issue with srid always 4326

### PR DESCRIPTION
**Pull request by achapkowski**
Fixes issue where data that is not projected in 4326 is handled properly.

### CODE REVIEW

File: geomet/esri.py [hunk 1]
```diff
@@ -160,7 +164,8 @@ def _dump_geojson_polygon(data, srid=None):
                 part_item.append(coord)
         if part_item:
             part_list.append(part_item)
-    srid = _extract_geojson_srid(data) or srid
+    if srid is None:
+        srid = _extract_geojson_srid(obj) or srid
```
Comment (larsbutler) [inline geomet/esri.py:168]: `obj` should be replaced with `data` on this line. This is what's causing the tests to fail.

**Comment by larsbutler** [comment]
@achapkowski Thanks for the patch. Are there any tests that could be added to trigger this condition?

File: geomet/esri.py (hunk 1 above)
Comment (achapkowski) [reply geomet/esri.py]: fixed.

**Comment by larsbutler** [comment]
@achapkowski Is there a sample input which breaks the code (prior to this change)? We should have at least 1 test case for this. I can write the tests if you can provide a sample input which triggers the bug.

**Comment by achapkowski** [comment]
sure thing, sorry I got caught up with life

**Comment by achapkowski** [comment]
i think we can close this, I lost the issue that I had and can no longer reproduce it.

**Pull request closed without merging**